supyr_struct/buffer.py

```diff
@@ -358,8 +404,11 @@ def close(self):
         # resized, but its in-memory buffer is still cleared
         mmap.close(self)
         try:
+            # TODO: This will always cause a ValueError according to
+            # https://docs.python.org/3/library/mmap.html#mmap.mmap.close
+            # Why are we doing this?
             self.clear_cache()
-        except Exception:
```

Comment:
I can't remember the exact reason, but looking at the cpython module, it should definitely always raise an exception.
https://github.com/python/cpython/blob/3.4/Modules/mmapmodule.c#L145

You can delete it.